migrate/src/main/scala/migrate/Main.scala

```diff
@@ -28,18 +30,19 @@ object Main {
     scala3ClassDirectory: AbsolutePath
   ): Try[Map[AbsolutePath, FileMigrationState.FinalState]] = {
     scribe.info(s"""Migrating
-                   |${sources.toList.mkString("\n")}""".stripMargin)
+                   |${unmanagedSources.toList.mkString("\n")}""".stripMargin)
     for {
       compiler <- setupScala3Compiler(scala3Classpath, scala3ClassDirectory, scala3CompilerOptions)
       initialFileToMigrate <-
-        buildMigrationFiles(sources, scala2Classpath, targetRoot, toolClasspath, scala2CompilerOptions)
-      _            <- compileInScala3(initialFileToMigrate, compiler)
+        buildMigrationFiles(unmanagedSources, scala2Classpath, targetRoot, toolClasspath, scala2CompilerOptions)
```

Comment:
I guess that Scalafix cannot find the patches in `HelloWorld` because it does not know that there is a  `Simple` class.

It is better if we don't give the `Simple.scala` file to Scalfix because we dont need the patches in that file. So maybe the `scala2Classpath` is not well configured (do we have the `target/scala-2.12/...` folder inside it)? Or maybe the semanticdb files are missing for `BuildInfo.scala`

Reply:
good idea. we don't give the managed files to scalafix. But I will check if the classpath is correct ^^ 
I think the semanticdb file is supposed to be there, the command start by executing `compile`